.github/workflows/pg-migtests.yml

```diff
@@ -168,6 +168,10 @@ jobs:
       - name: "TEST: pg-basic-live-migration-test"
         run: migtests/scripts/live-migration-run-test.sh pg/basic-live-test
 
+      # case sensitive table names are not yet supported in live migration, to restricting test only to a few tables.
```

Comment:
Should we run this here or on Jenkins?

Reply:
Ideally all tests should be run on jenkins, and a subset on GH actions. I don't really have a strong opinion here. Since it's multiple schema, maybe better to leave it here?

Reply:
Hmm, fine to me as well to just have a normal live migration test for this.